### DISCUSSION TITLE
Is it possible to use this tools to know more info related to person?

### DISCUSSION THREAD

**Discussion by Victor1890** [vladmandic/human#477] [General]
Hi there, I was to use this project to make one Face ID, but I think, is it possible to use this tools to know more info related to person?.
For example, I'm using glass, bracelets, cap, to know the background color, etc.

**Comment by vladmandic** [comment]
info as in description? you'd probably be better off using a VLM model for that, for example https://huggingface.co/microsoft/Florence-2-base-ft is small enough to run on endpoints and quite precise with descriptions.

**Reply by Victor1890** [reply]
Thank you so much @vladmandic I appreciate.👌
